### PR TITLE
Added properties as format

### PR DESCRIPTION
**Pull request by Jenson3210**
### Desired Outcome

We want to add properties format for liberty bootstrap.properties import support

### Implemented Changes

dotenv files do not support '.' like property files do. We added a new format 'properties' to support this 

### Definition of Done
*At least 1 todo must be completed in the sections below for the PR to be
merged.*

#### Changelog

- [X] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [X] This PR includes new unit and integration tests to go with the code
  changes, or
- [ ] The changes in this PR do not require tests

#### Documentation

- [X] Docs (e.g. `README`s) were updated in this PR
- [] A follow-up issue to update official docs has been filed here: [insert issue ID]
- [ ] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [ ] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [ ] There are no security aspects to these changes

### CODE REVIEW

**Comment by micahlee** [comment]
Hey @Jenson3210, thanks for contributing this change for properties validation! We're taking a look and we'll follow up shortly.

File: pkg/secrets/pushtofile/standard_templates_test.go [hunk 1]
```diff
@@ -221,6 +232,43 @@ func testValidateAliasLenForJSON(t *testing.T) {
 	}
 }
 
+func testValidateAliasForProperties(t *testing.T) {
```
Comment (micahlee) [inline pkg/secrets/pushtofile/standard_templates_test.go:235]: In order for these tests to run, this function needs to be upcased:

```suggestion
func TestValidateAliasForProperties(t *testing.T) {
```

File: design/m1_push_to_file_design.md [hunk 2]
```diff
@@ -161,13 +161,13 @@ format, for which there will be one secrets file per Conjur secret.
...
+| Annotation                                     | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```
Comment (micahlee) [inline design/m1_push_to_file_design.md:164]: We generally to not update design files after the development effort for that design is finished. Updating the documentation in `PUSH_TO_FILE.md` is great! Updating this design suggests that `properties` was in the original design, when it was not.

**Review by micahlee — COMMENTED** [review]
Hi @Jenson3210, thanks again for contributing this update for the `properties` template! 

This was a solid PR and I appreciate you taking the time to add the documentation and tests for it. With the way this project is set up, we've pulled the changes into an internal copy of this repository and have merged them there. That commit will show up here at the next release.

I've left a couple of comments just as feedback for future contributions. We've made these tweaks on the internal PR and you'll see them reflected in the commit once it's published.

Let me know if you have any questions!

File: design/m1_push_to_file_design.md (hunk 2 above)
Comment (Jenson3210) [reply design/m1_push_to_file_design.md:164]: Couldn't agree more 

Missed the fact that this is design up front. Just searched for all occurences in markdown. 😉

**Comment by Jenson3210** [comment]
Thanks for the feedback (much appreciated) + considerations.  

As this is not the source for the real images, feel free to close!

As this change got accepted, I will watch for the latest image to become available and we can happily continue setting up our usage in a optimal way. 

🙏

**Comment by Jenson3210** [comment]
Hi, 
Just to come back: we've tested the `edge` image published yesterday in our environment and the properties file is successfully generated. 
Looking forward to `latest`!

Thanks!

**Comment by Jenson3210** [comment]
@ismarc , @micahlee 

When we are running with the edge image to test, the files seem to be in a wrong template. 
Can you verify this once? 

We're seeing: 
"property.name": "secret.value"

2 things: 
- the key should not be surrounded by "" for a properties file, i think
- the `:` should be a `=`

**Pull request closed without merging**